cylc/flow/config_schema.py

```diff
@@ -0,0 +1,660 @@
+        'disable automatic shutdown': [VDR.V_BOOLEAN],
+        'simulation': {
+            'disable suite event handlers': [VDR.V_BOOLEAN, True],
+        },
+        'environment': {
+            '__MANY__': [VDR.V_STRING],
+        },
+        'parameters': {
+            '__MANY__': [VDR.V_PARAMETER_LIST],
+        },
+        'parameter templates': {
+            '__MANY__': [VDR.V_STRING],
+        },
+        'events': {
+            'handlers': [VDR.V_STRING_LIST, None],
+            'handler events': [VDR.V_STRING_LIST, None],
+            'startup handler': [VDR.V_STRING_LIST, None],
+            'timeout handler': [VDR.V_STRING_LIST, None],
+            'inactivity handler': [VDR.V_STRING_LIST, None],
+            'shutdown handler': [VDR.V_STRING_LIST, None],
+            'aborted handler': [VDR.V_STRING_LIST, None],
+            'stalled handler': [VDR.V_STRING_LIST, None],
+            'timeout': [VDR.V_INTERVAL],
+            'inactivity': [VDR.V_INTERVAL],
+            'abort if startup handler fails': [VDR.V_BOOLEAN],
+            'abort if shutdown handler fails': [VDR.V_BOOLEAN],
+            'abort if timeout handler fails': [VDR.V_BOOLEAN],
+            'abort if inactivity handler fails': [VDR.V_BOOLEAN],
+            'abort if stalled handler fails': [VDR.V_BOOLEAN],
+            'abort on stalled': [VDR.V_BOOLEAN],
+            'abort on timeout': [VDR.V_BOOLEAN],
+            'abort on inactivity': [VDR.V_BOOLEAN],
+            'mail events': [VDR.V_STRING_LIST, None],
+            'mail from': [VDR.V_STRING],
+            'mail smtp': [VDR.V_STRING],
+            'mail to': [VDR.V_STRING],
+            'mail footer': [VDR.V_STRING],
+        },
+        'reference test': {
+            'suite shutdown event handler': [
+                VDR.V_STRING, 'cylc hook check-triggering'],
+            'required run mode': [
+                VDR.V_STRING,
+                '', 'live', 'simulation', 'dummy-local', 'dummy'],
+            'allow task failures': [VDR.V_BOOLEAN],
+            'expected task failures': [VDR.V_STRING_LIST],
+            'live mode suite timeout': [
+                VDR.V_INTERVAL, DurationFloat(60)],
+            'dummy mode suite timeout': [
+                VDR.V_INTERVAL, DurationFloat(60)],
+            'dummy-local mode suite timeout': [
+                VDR.V_INTERVAL, DurationFloat(60)],
+            'simulation mode suite timeout': [
+                VDR.V_INTERVAL, DurationFloat(60)],
+        },
+        'authorization': {
+            # Allow owners to grant public shutdown rights at the most, not
+            # full control.
+            'public': (
+                [VDR.V_STRING, '']
+                + [level.name.lower().replace('_', '-') for level in [
+                    Priv.IDENTITY, Priv.DESCRIPTION, Priv.STATE_TOTALS,
+                    Priv.READ, Priv.SHUTDOWN]])
+        },
+    },
+    'document viewers': {
+        'html': [VDR.V_STRING, 'firefox'],
+    },
```

Comment:
Should move to separate client-only configuration file?

Reply:
Perhaps. I thought that the idea was to streamline things though.

Reply:
Yes, or just hard code. The online doc URL is unlikely to change during the life time of the release.

Reply:
We probably don't need this configuration now anyway?

Reply:
Yeah, get rid of this item. It dates back to pre web site days, when locally-installed documentation (not even behind a web server) we de rigeur.